### DISCUSSION TITLE
Cannot set up and run the project on my local device

### DISCUSSION THREAD

**Discussion by MihirRajChowdhury** [sButtons/sbuttons#1625] [Q&A]
Sorry though you may find me dumb but this is my first time contributing to project and I know how to fork the project and clone it but i don't know how to set it up on my local device I tried reading the developers guide but I was still unable to set it up this is the list of things that I did from the start while setting up the file 
1. git clone 
2. cd sbuttons
3. cd website 
4. npm install
5. npm start
6. npm run compile 
but it doesn't start the project on my device can anyone please help me on how do i set it up on my local device please because I really want to contribute to open source please

> sbuttons@1.9.6 start
> npm run watch:less

> sbuttons@1.9.6 watch:less
> onchange -v src/**/*.less -- npm run compile

onchange: watcher ready
this is what it is showing after i am running the npm start command

**Comment by johnnydomingo** [comment]
Hello, 
I am exactly in the same position as you, hoping to contribute to my first open source project. I just forked the repo and I will be going through the steps. If I find success, I will reach back out to you. If someone else is able to offer insight, it would be greatly appreciated!

**Comment by rebekahsnow** [comment]
I am getting the same issue, I would love to contribute to the project but I need to get this to work!

**Comment by shahednasser** [comment]
Hello everyone @MihirRajChowdhury @johnnydomingo @rebekahsnow 

Can anyone confirm whether they resolved this issue? I can't seem to understand what the problem is.

Looking at the first question, @MihirRajChowdhury are you running `npm run compile` inside website or in the root directory?
